### PR TITLE
Lambda: simple lambda custom responses

### PR DESCRIPTION
**Pull request by rafcio19**
Customise Lambda response by posting desired results to a new, static moto endpoint.

Resolves https://github.com/getmoto/moto/issues/7481

### CODE REVIEW

File: moto/awslambda_simple/models.py [hunk 1]
```diff
@@ -20,9 +24,10 @@ def invoke(
         response_headers: Any,
     ) -> Optional[Union[str, bytes]]:
 
-        if body:
-            return str.encode(body)
-        return b"Simple Lambda happy path OK"
+        default_result = ""
```
Comment (bblommers) [inline moto/awslambda_simple/models.py:27]:
```suggestion
        default_result = str.encode(body) if body else b"Simple Lambda happy path OK"
```

No reason to change that for existing  users - I'm sure someone's tests depend on that specific output :slightly_smiling_face:

Comment (bblommers) [reply moto/awslambda_simple/models.py:27]: I agree that an empty string makes a lot more sense, btw - but it is technically a breaking change, so we should only do that as part of a major release.

Comment (rafcio19) [reply moto/awslambda_simple/models.py:27]: Good point but the rationale is: this more accurately reflects what happens with real lambda in AWS, by default nothing is returned.

Happy to change this back though 🤷

**Review by bblommers — APPROVED** [review]
LGTM - thank you @rafcio19!

**Pull request merged**